src/Compilers/Core/Portable/Operations/IFlowCaptureReferenceOperation.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Operations
+{
+    /// <summary>
+    /// Represents a point of use of an intermediate result captured earlier.
```

Comment:
Consider mentioning `IFlowCaptureOperation` in the doc comment here. Should we also mention that these operations are generated only while generating a control flow graph?

Reply:
> Consider mentioning IFlowCaptureOperation in the doc comment here. Should we also mention that these operations are generated only while generating a control flow graph?

Will add this information in the next PR

---
In reply to: [163291820](https://github.com/dotnet/roslyn/pull/24358#discussion_r163291820) [](ancestors = 163291820)